### DISCUSSION TITLE
Using different retention schedules

### DISCUSSION THREAD

**Discussion by a-hersa** [offen/docker-volume-backup#492] [Q&A] [answered]
Hi,

I am following the documentation to add three diferent retention schedules but I am unsure about how to do it. It says "you could create three configuration files and mount them into `/etc/dockervolumebackup/conf.d`" but, how is that done?

Many thanks.

**Comment by m90** [comment, marked as answer]
As show in the example here https://offen.github.io/docker-volume-backup/how-tos/define-different-retention-schedules.html you create one `.conf` file for each schedule and then mount these into the `/etc/dockervolumebackup/conf.d` directory so the tool picks up the configuration from there. There's further info on that here https://offen.github.io/docker-volume-backup/how-tos/run-multiple-schedules.html

**Comment by a-hersa** [comment]
Thanks! I didn't see the second "how-to" I needed to understand it all. I want to backup a postgres db. Would not stopping the container before the backup corrupt the data? This way it would not affect the "run time".

**Reply by m90** [reply]
It's hard to tell without knowing the specifics of your application. You _might_ get away with a working backup even when not stopping the database, however there's no guarantee.

If you don't want to stop your database, you can use an approach like shown here https://offen.github.io/docker-volume-backup/recipes/#using-mysqldump-to-prepare-the-backup where you create a dump while the database is running, and then backup the dump instead of the actual data. This would work while the database is up 24/7 and also give you well formed backups always.

**Reply by a-hersa** [reply]
It seems it takes the same amount of time to set it up the safe way. I really appreciate your time and apologies if it was already in the docs. It's easy for non-techs to get lost in those and freeze.
